model_dealership.rb

```diff
@@ -0,0 +1,68 @@
+require 'csv'
+
+module CarLoader
+  def self.get_cars_from_csv(filepath)
+    @cars = CSV.read(filepath, { :headers => true, :header_converters => :symbol}).map { |car_object| Car.new(car_object) }
+  end
+end
+
+class Car
+  include CarLoader
```

Comment:
This does not need to be included here since you do not need the Car class to have the CarLoader's methods. You are calling the CarLoader module's method's outside of the car class (in the global scope before you instantiate the dealership).